src/com/reason/ide/files/DuneFileType.java

```diff
@@ -10,8 +10,13 @@
 import javax.swing.*;
 
 public class DuneFileType extends LanguageFileType {
+
     public static final FileType INSTANCE = new DuneFileType();
 
+    public static String getDefaultFilename() {
```

Comment:
@giraud please see this PR: https://github.com/reasonml-editor/reasonml-idea-plugin/pull/226